src/log.js

```diff
@@ -548,6 +548,19 @@ export function rethrowAsync(var_args) {
 }
 
 
+/**
+ * Rethrows the error immediately. Required because async errors thrown during
+ * tests leak to subsequent tests. DO NOT USE IN RUNTIME CODE.
+ * @visibleForTesting
+ * @param {...*} var_args
+ */
+export function rethrowAsyncForTests(var_args) {
```

Comment:
we allow rest/spread in the code base right? lets just use that instead of var_args

Reply:
We use `var_args` in several other spots in `log.js`, so I wanted to be consistent here. Also, there's a lint rule against using the spread operator.

https://github.com/ampproject/amphtml/blob/db751cb6e9db35cda235484cf6e8f5708a41566b/.eslintrc#L74

If it's possible to use rest/spread here, I think it makes sense to switch this and all other instances in the file in a separate PR.